src/main/java/com/iexec/worker/result/ResultService.java

```diff
@@ -272,12 +276,13 @@ String getWeb2ResultLink(String chainTaskId) {
 
         switch (storage) {
             case IPFS_RESULT_STORAGE_PROVIDER:
-                String ipfsHash = customResultFeignClient.getIpfsHashForTask(chainTaskId);
-                if (ipfsHash.isEmpty()) {
+                try {
+                    String ipfsHash = resultProxyClient.getIpfsHashForTask(chainTaskId);
+                    location = "/ipfs/" + ipfsHash;
+                } catch (RuntimeException e) {
                     log.error("Cannot get tee web2 result link (result-proxy issue) [chainTaskId:{}]", chainTaskId);
```

Comment:
```suggestion
                    log.error("Cannot get tee web2 result link (result-proxy issue) [chainTaskId:{}]", chainTaskId, e);
```

Reply:
I will do it right now.
We had a discussion some time ago on this signature which is not explicitly documented.
But it is supported and I finally found the official answer here :
https://www.slf4j.org/faq.html#paramException

Reply:
Fixed in a11dccb